assets/style.css

```diff
@@ -0,0 +1,505 @@
+
+*,
+::before,
+::after {
+  box-sizing: border-box;
+  font-family: 'Lato', sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  background-color: #f8f5f5;
+  min-height: 100vh;
+}
+
+.navbar {
+  display: none;
+}
+
+.hamburger {
+  width: 25px;
+  height: 25px;
+  margin-bottom: 10%;
+}
+
+/* intro section */
+.intro {
+  background-image: url('./images/header.jpg');
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover;
+}
+
+.blur {
+  display: flex;
+  flex-direction: column;
+  gap: 1%;
+  padding-left: 2%;
+  padding-right: 3%;
+  padding-top: 3%;
+  padding-bottom: 10%;
+  backdrop-filter: blur(5px);
+  background-color: rgba(255, 255, 255, 0.3);
+}
+
+.welcome {
+  font-size: large;
+  font-weight: bold;
+  letter-spacing: 1px;
+  font-style: italic;
+  color: #5a3f3b;
+}
+
+.title {
+  font-size: xx-large;
+  font-weight: bold;
+  color: #8f6b6b;
+}
```

Comment:
- [ ] You did a fantastic job adding the main title to your index page :+1:, but to match with the template I think it would be better if you make the title a little bit bigger and also use a greater `font-weight` :1st_place_medal: